emitter/src/tests/action.spec.ts

```diff
@@ -225,4 +242,28 @@ describe('Actions', () => {
     fixture.componentInstance.decrement!.emit();
     fixture.componentInstance.multiplyBy2!.emit();
   });
+
+  it('should dispatch actions to correct receivers', () => {
+    TestBed.configureTestingModule({
+      imports: [NgxsModule.forRoot([CounterState]), NgxsEmitPluginModule.forRoot()],
+      declarations: [MockComponent]
+    });
+
+    const store: Store = TestBed.get(Store);
+    const actions$: Actions = TestBed.get(Actions);
+
+    const action = actions$
+      .pipe(
+        ofEmittableSuccessful(CounterState.dontChange, CounterState.increment, CounterState.decrement),
+        take(3)
+      )
+      .toPromise()
+      .then(() => {
+        const counter = store.selectSnapshot<number>((state) => state.counter);
+        expect(counter).toBe(0);
+      });
+
+    CounterStore.dontChange.emit();
```

Comment:
This would be:
```ts
CounterState.dongChange.emit();
expect(...).to(...);
```

Reply:
Again, this can only be applied to synchronous test. Actions are dispatched asynchronously so I have to write assertions in callback.

Reply:
Actions are NOT dispatched asynchronously.